### ISSUE TITLE
Error: This is not a supported input!

### ISSUE THREAD

**Issue by caiootto** [platomav/BIOSUtilities#24]
Hello!
I am trying to extract the .bin file with the Dell_PFS_Extract script and it is resulting in this message "Error: This is not a supported input!".
Does it really fail to extract the file of this type of .exe or am I doing something wrong? I'll put a screenshot to see if it can help me. The .exe is a dell notebook bios that I downloaded from the dell website.
https://www.dell.com/support/home/pt-br/drivers/driversdetails?driverid=5rnv1&oscode=wt64a&productcode=inspiron-14-5442-laptop
Thanks for help me.

![Dell PFS](https://user-images.githubusercontent.com/75398096/224120848-444b328f-6c52-4834-aab5-a84841e9fd16.jpg)

**Comment by platomav** [comment]
That error message indicates that the input file is not a format that is applicable/relevant to the utility. You'll need to try some of the other utilities of the repo, in case one of them is applicable to the file format you're trying to process. If nothing works, it's not a file format supported by the utilities of this repo, and you may find help by asking at forums such as [Win-Raid](https://winraid.level1techs.com/), [BIOS-Mods](https://www.bios-mods.com/forum/), [MyDigitalLife ](https://forums.mydigitallife.net/) etc.

**Comment by caiootto** [comment]
I managed to solve
This bios is from Insyde so I tested it with the Insyde_IFD_Extract.py script and it worked. I already tested the bios and it worked perfectly.
I'll send the prints as I managed to solve it.
Thank you very much for your attention and congratulations on the excellent work.

![Insyde_IFD_Extract 2](https://user-images.githubusercontent.com/75398096/224157319-a04386c2-3196-46be-947c-dabefff21c32.jpg)
![Insyde_IFD_Extract 3](https://user-images.githubusercontent.com/75398096/224157324-e2ed2286-1590-440d-840e-7707647203f7.jpg)
![Insyde_IFD_Extract](https://user-images.githubusercontent.com/75398096/224157328-3f0af470-6cb1-4ea4-8fa0-290afdb5a89f.jpg)
